apriltags_ros/src/apriltag_detector.cpp

```diff
@@ -68,7 +76,16 @@ AprilTagDetector::~AprilTagDetector(){
   image_sub_.shutdown();
 }
 
+void AprilTagDetector::enableCb(const std_msgs::Bool& msg) {
+  enabled_ = msg.data;
```

Comment:
Because it is a nodelet, I don't think it is actually serializing the image.  Instead, it just passes a pointer in, so it probably isn't an issue in this case?  That said, I can unsubscribe to be sure.